odl/tomo/backends/astra_cuda.py

```diff
@@ -169,6 +177,9 @@ def __del__(self):
             aproj.delete(self.proj_id)
             self.proj_id = None
 
+        self.in_array = None
```

Comment:
These are no longer needed inside `__del__` since they dissapear automatically by garbage collection. Remove in both places.